### PR TITLE
Bring Travis and AppVeyor configurations in sync with the GLib one

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
👍

File: .travis.yml [hunk 1]
```diff
@@ -54,15 +54,11 @@ script:
     make regen_check;
     fi
   - ./check_init_asserts
```
Comment (sdroege) [inline .travis.yml]: What should we do about
```
$ ./check_init_asserts
src/event.rs:set_handler(ptr: glib_ffi::gpointer) 
The command "./check_init_asserts" exited with 1.
```
?

Comment (EPashkin) [reply .travis.yml]: Maybe copy script from gtk?

**Comment by sdroege** [comment]
@GuillaumeGomez all green now

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**